include/wx/arrstr.h

```diff
@@ -42,12 +42,30 @@ wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2)
     return cmp ? cmp : s1.Cmp(s2);
 }
 
+
 inline int wxCMPFUNC_CONV
 wxDictionaryStringSortDescending(const wxString& s1, const wxString& s2)
 {
     return wxDictionaryStringSortAscending(s2, s1);
 }
 
+WXDLLIMPEXP_BASE
+int wxCMPFUNC_CONV wxCmpNatural(const wxString& s1, const wxString& s2);
+
+WXDLLIMPEXP_BASE
+int wxCMPFUNC_CONV wxCmpGenericNatural(const wxString& s1, const wxString& s2);
```

Comment:
Nitpicking, but I wonder if this shouldn't be rather called `wxCmpNaturalGeneric()` as we seem to be putting `Generic` either in the beginning or at the end, but rarely in the middle AFAICS.

Reply:
Makes sense, my intention was probably to maintain wxCmp prefix to indicate to which function family it belongs and the generic classes AFAIK start with "Generic", i.e., `wxGenericDirCtrl`.

Will change to your suggestion.